src/views/splash.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Hack Cambridge Ternary</title>
+        <link rel="stylesheet" type="text/css" href="{{ asset('styles/hc-2018/splash-bundle.css') }}">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskRegular-Regular.css">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskScreen-Regular.css">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskLight-Regular.css">
+        <link rel="stylesheet" type="text/css" href="https://assets.hackcambridge.com/fonts/PxGroteskBold-Regular.css">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.png') }}" sizes="512x512" type="image/png">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.ico') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256" type="image/vnd.microsoft.icon">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.icns') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256 512x512">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.svg') }}" sizes="any" type="image/svg+xml">
+        <meta name=description content="Get hyped for Hack Cambridge 2018">
+        <meta name=keywords content="hack,Cambridge,ternary,sponsorship,hackathon">
+        <meta name=author content="Hack Cambridge Team">
+        <meta name=viewport content="width=device-width, initial-scale=1.0">
+    </head>
```

Comment:
There's quite a lot of stuff common to the splash page and the sponsorship page here in both the header and the footer.  I suggest we use a base template like base.html from last year to make it easier to change this stuff.

Reply:
I've refactored everything in `<head>` into `base.html`. Last year you could only set the `title` individually for each page—we can now set `description` and `keywords` too.
The `<footer>` can't be refactored wholesale into `footer.html` because in some cases (such as the splash page) it needs to be in a column whose size is determined by the rest of the page. So in `footer.html` you'll find the _innermost contents_ of `<footer>`.